dogfood/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
```

Comment:
Missing new line. We should add an .editorconfig file

Reply:
since I don't use .editorconfig files, I'll leave that for another PR.